tests/draft2019-09/patternProperties.json

```diff
@@ -155,7 +155,8 @@
                 "^🐲*$": {
                     "type": "integer"
                 }
-            }
+            },
+            "additionalProperties": false
```

Comment:
(These additionalProperties are stray now though.)

Reply:
> I again don't like it that we have an entire set of tests with no negative cases.
> 
> What was wrong with the suggestion I made? Do you disagree with it?

@Julian Nope, I totally agree with you, but how do I make a negative case without the use of "additionalProperties"? For some reason, the solution isn't coming to me...